lib/tasks/test.js

```diff
@@ -181,7 +184,7 @@ module.exports.browserTest = function(gulp, config) {
 	return Promise.all([
 		process.env.HEROKU_AUTH_TOKEN ? Promise.resolve(process.env.HEROKU_AUTH_TOKEN) : commandLine.run('heroku', ['auth:token']),
 		commandLine.run('git', ['rev-parse HEAD']),
```

Comment:
Not part of the PR so don't worry too much, but it might be better to be consistent here. Instead of space separating arguments, having them as separate entries into the array.